spec/features/shelters/user_can_create_shelter_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "Create shelter page", type: :feature do
+    it "can access shelter creation page" do
+        visit '/shelters'
+        click_link("New Shelter")
+        fill_in "shelter[name]", with: "Dog Town"
+        fill_in "shelter[address]", with: "1 Club Drive"
+        fill_in "shelter[city]", with: "Ken Caryl"
+        fill_in "shelter[state]", with: "CO"
+        fill_in "shelter[zip]", with: "80123"
+        click_button "submit"
```

Comment:
All of this is to check that the link works and then it fills out the form with shelter information.